customize: Minor fixes (#206)
inveniosoftware/docs-invenio-rdm: docs/customize/application_data.md

  allow: <array of strings>
```

> - `<email>` : email of the user.

Reviewer: If we add a period at the end, might as well capitalize the first letter.

To load vocabularies, the typical file structure is the one depicted above:

> - a `vocabularies.yaml` files that contains the specific vocabularies that you want to override.
> - a `vocabularies/<vocabulary_identifier>.yaml` file that contains an entry for each vocabulary type.

Reviewer: ```suggestion
- a `vocabularies.yaml` file that contains the specific vocabularies that you want to override.
- `vocabularies/<vocabulary_identifier>.yaml` files that contain entries for each vocabulary or vocabulary scheme.
```

vocabulary is e.g. `subjects`
vocabulary type / scheme / subtype is e.g. `MeSH`
vocabulary record is a full MeSH record content.

Maybe we can place multiple `<vocabulary_identifier>.yaml` in the tree above to make it even clearer.
So 
```
